src/model/bot.py

```diff
@@ -265,28 +320,78 @@ def drop_all(self, skip_rows: int = 0, skip_slots: List[int] = None) -> None:
         pag.keyUp("shift")
 
     def drop(self, slots: List[int]) -> None:
+        self.drop(slots, DropOrder.HORIZONTAL, False)
+
+    def drop(self, slots: List[int], drop_order: DropOrder, misclick_chance: bool) -> None:
         """
-        Shift-clicks inventory slots to drop items.
+        Shift-clicks inventory slots to drop items. 
         Args:
             slots: The indices of slots to drop.
-        """
+            drop_order: The overall order in which to try dropping slots
+            misclick_chance: If True provides a chance that a slot will not be dropped on the first pass.
+                             If a slot is missed then it will be reattempted after all the others have been attempted.
+                             Offers more human-like behavior if set.
+        """
+        slots_to_drop = slots
```

Comment:
Pretty nitpicky, but this actually shallow copies the `slots` argument. Not sure if anyone would actually run into any issues but it is generally best practice to deep copy the arguments to ensure you don't modify the arguments passed in. 
For example, if someone wanted to be sure their inventory length was less than it was after the action is completed:
```
slots_before_drop = api_s.get_inv_item_indices(item_id) # let's say the len is 10
self.drop(slots_before_drop , DropOrder.HORIZONTAL_ALTERNATING, True)
slots_after_drop = api_s.get_inv_item_indices(item_id) # let's say the len is 0 now

#  len(slots_before_drop) will now be 0 since it was modified in Bot.drop on line 388 with `slots_to_drop.remove(i)`
if len(slots_after_drop) >= len(slots_before_drop):
   self.log_msg("Drop inventory was unsuccessful")
```
I am not too familiar with Python, but I believe the `deepcopy` method from the `copy` library would fix it. There may be more efficient ways of deep-copying a list though.

Reply:
Heck yeah, stoked for this type of functionality. Was thinking about making it since I definitely don't do the pattern currently used for dropping so I will definitely use this 👍